docs/rancher/virtualization-management.md

```diff
@@ -6,62 +6,62 @@ keywords:
 
 
-### Example
+### Multi-Tenancy Example
 The following example provides a good explanation of how the multi-tenant feature works:
 
-1. Firstly, add new users via the Rancher `Users & Authentication` page, click `Create` to add two new separated users e.g. the `project-owner` and `project-readonly` respectively.
-    - A `project-owner` is a user with permission to manage a list of resources of a particular project e.g, the default project.
-    - A `project-readonly` is a user with read-only permission of a particular project e.g, the default project.
+1. First, add new users via the Rancher `Users & Authentication` page. Then click `Create` to add two new separated users, such as `project-owner` and `project-readonly` respectively.
+    - A `project-owner` is a user with permission to manage a list of resources of a particular project, e.g., the default project.
+    - A `project-readonly` is a user with read-only permission of a particular project, e.g., the default project.
     ![](./assets/create-user.png)
-2. Click one of the imported Harvester cluster and visiting the Harvester UI
-    - click the `Projects/Namespaces` tab
-    - select a project e.g, `default` and click the `Edit Config` menu to assign the users to this project with appropriate permissions,  e.g. the `project-owner` user will be assigned with project owner role.
+1. Click one of the imported Harvester clusters after navigating to the Harvester UI.
+    - Click the `Projects/Namespaces` tab.
+    - Select a project such as `default` and click the `Edit Config` menu to assign the users to this project with appropriate permissions. For example, the `project-owner` user will be assigned the project owner role.
    ![](./assets/add-member.png)
-4. Continue to add the project-readonly user to the same project with read-only permissions and click save.
+1. Click to **Add** the `project-readonly` user to the same project with read-only permissions and click **Save**.
    ![](./assets/added-user.png)
-5. Open an incognito Browser and login to different accounts e.g, the `project-owner`
-6. After login with the `project-owner` user, Click the Virtualization Management menu, you should be able to view the cluster you have been assigned to.
-7. Click the Images tab, you should be able to view a list of images uploaded to the harvester-public namespace previously, or you can upload your own image if needed.
-8. Create a VM with one of the images that you have uploaded
-9. Login with another user e.g, `project-readonly`, and this user will only have the read permission of this project.
+1. Open an incognito browser and log in as `project-owner`.
+1. After logging in as the `project-owner` user, click the **Virtualization Management** tab. There you should be able to view the cluster to which you have been assigned.
+1. Click the **Images** tab to view a list of images previously uploaded to the Harvester-public namespace. You can also upload your own image if needed.
```

Comment:
I think `harvester-public` is the actual namespace so it doesn't need to be capitalized.